### PR TITLE
Dark Mode: Settings Search Bar

### PR DESCRIPTION
**Pull request by darkwing**
MoreSearch
SettingsSearcher

### CODE REVIEW

**Review by GuillaumeRx — CHANGES_REQUESTED** [review]

File: ui/pages/settings/index.scss [hunk 1]
```diff
@@ -68,7 +68,7 @@
       }
 
       &__list {
-        background: var(--color-background-default);
```
[the remark this answers is not in the document]
Comment (darkwing) [reply ui/pages/settings/index.scss:71]: This must've been a rebase conflict, my mistake!

File: ui/pages/settings/settings-search/settings-search.js [hunk 2]
```diff
@@ -15,7 +15,9 @@ export default function SettingsSearch({
   const t = useContext(I18nContext);
 
   const [searchQuery, setSearchQuery] = useState('');
-  const [searchIconColor, setSearchIconColor] = useState('#9b9b9b');
+  const [searchIconColor, setSearchIconColor] = useState(
+    'var(--color-icon-default)',
```
Comment (GuillaumeRx) [inline ui/pages/settings/settings-search/settings-search.js:19]:
```suggestion
    'var(--color-icon-muted)',
```

File: ui/pages/settings/settings-search/settings-search.js [hunk 3]
```diff
@@ -32,9 +34,9 @@ export default function SettingsSearch({
   const handleSearch = (searchQuery) => {
     setSearchQuery(searchQuery);
     if (searchQuery === '') {
-      setSearchIconColor('#9b9b9b');
+      setSearchIconColor('var(--color-icon-muted)');
     } else {
-      setSearchIconColor('#24292E');
+      setSearchIconColor('var(--color-text-default)');
```
Comment (GuillaumeRx) [inline ui/pages/settings/settings-search/settings-search.js:39]:
```suggestion
      setSearchIconColor('var(--color-icon-default)');
```

**Review by GuillaumeRx — CHANGES_REQUESTED** [review]

**Review by GuillaumeRx — APPROVED** [review]

**Review by georgewrmarshall — APPROVED** [review]
lgtm

**Pull request merged**